sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/ImageDataFetcher.cs

```diff
@@ -32,17 +32,17 @@ public override void FetchData(DataFetcherParameters parameters)
 			imageDataParameters.tile.AddTile(rasterTile);
 		}
```

Comment:
@atripathi-mb I think you mean moving `vectorTile.Initialize` into the null check? 
that would mean we can't do any requests without a tile object and that's what I was trying to avoid here. Developers using this class should be able to make requests with the basic parameters, without having to use `UnityTile` class and object.
At this moment, they can call this class with just the required parameters from their custom classes.